src/goose.rs

```diff
@@ -1200,10 +1200,7 @@ impl GooseClient {
             match &response {
                 Ok(r) => {
                     let status_code = r.status();
-                    // Only increment status_code_counts if we're displaying the results
-                    if self.config.status_codes {
-                        raw_request.set_status_code(Some(status_code));
-                    }
+                    raw_request.set_status_code(Some(status_code));
```

Comment:
STRICT mode on - the tests PR should only have the changes in tests.

Any other changes should be made outside in an extra PR.

Reply:
This was required for the response to correctly include the status code. It was a lingering bug from when we used to process statistics in real-time, but now we must always store the status code.